### PR TITLE
CIV-11542 Bilingual claim creation

### PR DESCRIPTION
**Pull request by ElifDenizEsea**
**Before creating a pull request make sure that:**

- [x] commit messages are meaningful and follow good commit message guidelines
- [x] README and other documentation has been updated / added (if needed)
- [x] tests have been updated / new tests has been added (if needed)

### JIRA link (if applicable) ###

https://tools.hmcts.net/jira/browse/CIV-11542

### Change description ###

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x] No
```

### CODE REVIEW

**Review by neeta-hmcts — DISMISSED** [review]

File: src/main/java/uk/gov/hmcts/reform/civil/model/citizenui/CcdDashboardClaimMatcher.java [hunk 1]
```diff
@@ -0,0 +1,27 @@
+package uk.gov.hmcts.reform.civil.model.citizenui;
+
+import lombok.AllArgsConstructor;
+import uk.gov.hmcts.reform.civil.model.CaseData;
+
+@AllArgsConstructor
+public class CcdDashboardClaimMatcher {
```
Comment (portizdev) [inline src/main/java/uk/gov/hmcts/reform/civil/model/citizenui/CcdDashboardClaimMatcher.java:7]: Why do you need this in your pr?

**Review by portizdev — CHANGES_REQUESTED** [review]

File: src/main/java/uk/gov/hmcts/reform/civil/model/CaseDataParent.java [hunk 2]
```diff
@@ -722,6 +722,7 @@ public boolean isApplicantNotRepresented() {
     private DynamicList transferCourtLocationList;
     private NotSuitableSdoOptions notSuitableSdoOptions;
     private TocTransferCaseReason tocTransferCaseReason;
+    private String claimantBilingualLanguagePreference;
```
Comment (manish14836) [inline src/main/java/uk/gov/hmcts/reform/civil/model/CaseDataParent.java:725]: Is it worth to change it to ENUM(ENGLISH and BOTH)?

File: src/main/java/uk/gov/hmcts/reform/cmc/model/FormaliseOption.java [hunk 3]
```diff
@@ -0,0 +1,16 @@
+package uk.gov.hmcts.reform.cmc.model;
+
```
Comment (manish14836) [inline src/main/java/uk/gov/hmcts/reform/cmc/model/FormaliseOption.java:2]: Is this class related to CIV-11542. cannot see any usage this is PR?

File: src/main/java/uk/gov/hmcts/reform/civil/model/citizenui/CcdDashboardClaimMatcher.java (hunk 1 above)
Comment (ElifDenizEsea) [reply src/main/java/uk/gov/hmcts/reform/civil/model/citizenui/CcdDashboardClaimMatcher.java]: merge problem fixed

File: src/main/java/uk/gov/hmcts/reform/civil/model/CaseDataParent.java (hunk 2 above)
Comment (ElifDenizEsea) [reply src/main/java/uk/gov/hmcts/reform/civil/model/CaseDataParent.java:725]: there is one related enum for Welsh, English , Welsh and English I did not wanted to add one more

File: src/main/java/uk/gov/hmcts/reform/cmc/model/FormaliseOption.java (hunk 3 above)
Comment (ElifDenizEsea) [reply src/main/java/uk/gov/hmcts/reform/cmc/model/FormaliseOption.java]: merge problem fixed

**Review by portizdev — APPROVED** [review]

**Review by kalachandrasekar1 — APPROVED** [review]

**Pull request merged**